tests/cases/fourslash/jsdocDeprecated_suggestion15.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference path="fourslash.ts" />
+
+// @module: esnext
+// @filename: /a.ts
+////export const a = 1;
+////export const b = 1;

```

Comment:
does this apply to same-file export lists too, like `export {a, /** @deprecated */ b }`? If so, we should have a test for that as well.

Reply:
I've added test.